action.yml

```diff
@@ -1,5 +1,9 @@
 name: "Conventional Commits"
 description: "Ensures that all commit messages are following the conventional-commits standard."
+inputs:
+  GITHUB_TOKEN:
+    description: 'GitHub token'
+    required: true
```

Comment:
Hm, do we want to mark this as required? 🤔 

I mean, the action worked without it, so I guess we should make this optional. Can we do that?

Reply:
set required = false, Pls check it out.